app/src/main/java/com/example/paritosh/stormy/WeatherPresenter.java

```diff
@@ -0,0 +1,43 @@
+package com.example.paritosh.stormy;
+
+import com.example.paritosh.stormy.model.CurrentWeather;
+import com.example.paritosh.stormy.model.CurrentWeatherDataBindingModel;
+
+public class WeatherPresenter {
+    WeatherActivity weatherActivity;
```

Comment:
Please make it private

Reply:
Done! Don't see a point of pushing the branch with 1 change. What say?